packages/create-snap/src/cli.test.ts

```diff
@@ -0,0 +1,78 @@
+import { cli } from './cli';
+import { init } from './cmds';
+
+const getMockArgv = (...args: string[]) => {
+  return ['/mock/path', '/mock/entry/path', ...args];
+};
+
+const HELP_TEXT_REGEX = /^\s*create-metamask-snap \[directory\]/u;
+
+describe('cli', () => {
+  let consoleLogSpy: jest.SpyInstance;
+  let processExitSpy: jest.SpyInstance;
+
+  beforeAll(() => {
+    global.snaps = {};
+  });
+
+  beforeEach(() => {
+    consoleLogSpy = jest.spyOn(console, 'log');
+    processExitSpy = jest.spyOn(process, 'exit');
+  });
+
+  afterAll(() => {
+    delete (global as any).snaps;
+  });
+
+  it('exits if no argument was provided', () => {
+    consoleLogSpy.mockImplementationOnce((message: string) => {
+      expect(message).toMatch(HELP_TEXT_REGEX);
+    });
+
+    processExitSpy.mockImplementationOnce(() => {
+      throw new Error('process exited');
+    });
+
+    expect(() => cli(getMockArgv('--help'))).toThrow('process exited');
+  });
+
+  it('calls "help" command', async () => {
```

Comment:
To ensure that all assertions in this test are called, something like this (at the top of the test) can be useful:

```ts
expect.assertions(2);
```

Reply:
Got it. I updated this in the `snaps-cli` test as well.